lib/bpf/common.go

```diff
@@ -117,6 +117,26 @@ func (c *Config) CheckAndSetDefaults() error {
 	return nil
 }
 
+// RestrictedSessConfig holds configuration for the RestrictedSession service.
+type RestrictedSessConfig struct {
```

Comment:
Spell it out? It's part of the public API.

```suggestion
// RestrictedSessionsConfig holds configuration for the RestrictedSession
// service.
type RestrictedSessionsConfig struct {
```